tooling/build/scripts/publisher.sh

```diff
@@ -106,7 +106,12 @@ ls -al
 # Publish to S3
 echo "Publishing to S3..."
 start_time=$(date +%s)
-aws s3 sync . s3://$S3_BUCKET_NAME/$SITE_NAME/$CODEBUILD_BUILD_NUMBER/latest --delete --no-progress
+
+# Set all files to have 10 minutes of cache, except for those in the _next folder
+aws s3 sync . s3://$S3_BUCKET_NAME/$SITE_NAME/$CODEBUILD_BUILD_NUMBER/latest --delete --no-progress --cache-control "max-age=600" --exclude "_next/*"
```

Comment:
technically each time this is run a new build folder is create, hence there is no impact of --delete flag right

Reply:
Yeah no impact, but this is a more defensive approach in case there is existing data.